tap_s3_csv/transform.py

```diff
@@ -125,6 +125,12 @@ def __enter__(self):
     def __exit__(self, *args):
         self.log_warning()
 
+    def cleanup(self):
```

Comment:
why don't you just put this in __exit__ and revert back to `with transform.Transformer(source_type_for_updatecol_map) as transformer:` in sync.py? add a flag for clearing in the constructor if you really need to maintain behavior for some reason